h5p-bildetema/language/da.json

```diff
@@ -0,0 +1,382 @@
+{
+  "semantics": [
+    {
+      "label": "Oversættelser",
+      "fields": [
+        {
+          "label": "Sprogkode",
+          "description": "Sprogkoden skal skrives med to tegn (engelsk: en, norsk bokmål: nb)",
```

Comment:
Totally unnecessary change

```suggestion
          "description": "Sprogkoden skal skrives med to tegn (engelsk: en, dansk: da)",
```

Reply:
It's only the english descriptions that are shown in WordPress actually, so not that necessary no.

<img width="703" alt="Skjermbilde 2022-11-29 kl  15 13 35" src="https://user-images.githubusercontent.com/35261194/204552218-048adfdf-0e6a-428f-9a28-d16d629454af.png">